arctic/tickstore/tickstore.py

```diff
@@ -636,6 +636,12 @@ def _ensure_supported_dtypes(array):
             array = array.astype('<f8')
         elif (array.dtype.kind) in ('U', 'S'):
             array = array.astype(np.unicode_)
+        elif (array.dtype.kind) == 'O':
+            try:
+                array = np.array([s.encode('ascii') for s in array])
+                array = array.astype(np.unicode_)
+            except:
+                raise UnhandledDtypeException("Unsupported dtype '%s' - only int64, float64 and U are supported" % array.dtype)
```

Comment:
Maybe add to the error that object columns must be coercible to strings?  This can be a little confusing for pandas users since 'U' can't actually be set cast to anymore.

Reply:
sure I can make the error message more specific for this specific failure